packages/language-server/test/snapshots/xml-view-diagnostics/use-of-deprecated-aggregation/input.xml

```diff
@@ -0,0 +1,8 @@
+<mvc:View xmlns:mvc="sap.ui.core.mvc" xmlns:charts="sap.ca.ui.charts">
+    <mvc:content>
+        <🢂charts:BubbleChart🢀>
```

Comment:
is there any agggregation which is deprecated yet the class it is inside of
is not deprecated?

A quick programmatic search on the module should be able to locate one.